contracts/Certificate.sol

```diff
@@ -98,7 +99,7 @@ contract Certificate is ERC1155PresetPausableNonTransferrable {
     // todo is there a better way to verify that no removal amount == 0?
     for (uint256 i = 0; i < removalIds.length; i++) {
       if (removalAmounts[i] == 0) {
-        revert("Certificate: Removal amount 0");
+        revert RemovalAmount0({removalId: removalIds[i]});
```

Comment:
this shouldn't be possible any more so we should just remove it

Reply:
great, I think we have a ticket now to help prevent the possibility of this further upstream